ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -151,10 +152,11 @@ func validateCommaSeparatedParam(cfnParams *cloudformation.CfnStackParams, param
 }
 
 func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecsclient.ECSClient, cfnClient cloudformation.CloudformationClient, amiIds ami.ECSAmiIds) error {
-	// Validate cli flags
-	if !isIAMAcknowledged(context) {
-		return fmt.Errorf("Please acknowledge that this command may create IAM resources with the '--%s' flag", command.CapabilityIAMFlag)
+	_, err := validateInstanceRole(context)
```

Comment:
Not really -- was sort of anticipating it being useful for future use, but can pare it back for now.